model/client.go

```diff
@@ -1735,6 +1735,30 @@ func (c *Client) GetTeamMembers(teamId string, offset int, limit int) (*Result,
 	}
 }
 
+// GetMyTeamMembers will return an array with team member objects that the current user
+// is a member of. Must be authenticated.
+func (c *Client) GetMyTeamMembers() (*Result, *AppError) {
+	if r, err := c.DoApiGet("/teams/members", "", ""); err != nil {
+		return nil, err
+	} else {
+		defer closeBody(r)
+		return &Result{r.Header.Get(HEADER_REQUEST_ID),
+			r.Header.Get(HEADER_ETAG_SERVER), TeamMembersFromJson(r.Body)}, nil
+	}
+}
+
+// GetMyTeamMembers will return an array with TeamMemberUnread objects that contain the amount of
+// unread messages and mentions the current user has for the teams it belongs to. Must be authenticated.
+func (c *Client) GetMyTeamMembersUnread() (*Result, *AppError) {
```

Comment:
`GetMyTeamMembersUnread` is a little confusing, maybe `GetUnreadsForTeam` or `GetMyUnreadsForTeam`?